css/style.css

```diff
@@ -1,14 +1,17 @@
 * {
     box-sizing: border-box;
     font-family: 'Titillium Web', sans-serif;
-    position: relative;
 }
 
-.index-body {
+html {
```

Comment:
Don't apply styles to the `html` tag; change this back to `.index-body` or a new container tag

Reply:
Actually, set it on the `body` tag - because that represents the visible html, which the `html` tag represents both the visible code and the metadata.

Reply:
putting it inside the html is the solution on this one sadly ( mike's word)